docs/config/dfdaemon_config_template.yml

```diff
@@ -49,28 +58,14 @@ dfget_flags: ["--node","192.168.33.21","--verbose","--ip","192.168.33.23",
               "--port","15001","--expiretime","3m0s","--alivetime","5m0s",
               "-f","filterParam1&filterParam2"]
 
-# Specify the addresses(host:port) of supernodes, it is just to be compatible with previous versions
-supernodes:
-    - 127.0.0.1
-    - 10.10.10.1
-
-# Net speed limit,format:xxxM/K
-ratelimit: 20M
-
-# Temp output dir of dfdaemon, it must be an absolute path. and the default value is `$HOME/.small-dragonfly/dfdaemon/data/`
-localrepo: /home/admin/.small-dragonfly/dfdaemon/data/
-
-# dfget path, which is the relative file path for the dfdaemon
-dfpath: ./dfget
-
-# https options
-# port: 12001
-# hostIp: 127.0.0.1
-# certpem: ""
-# keypem: ""
+# the temp output dir of dfget, it must be an absolute path.
+# default: `$HOME/.small-dragonfly/dfdaemon/data/`
+# localrepo: $HOME/.small-dragonfly/dfdaemon/data/
```

Comment:
emmm.`# default: `$HOME/.small-dragonfly/dfdaemon/data/`` look like the configuration which is annotated,how about 
```
the temp output dir of dfget, it must be an absolute path.the default value is $HOME/.small-dragonfly/dfdaemon/data/
# localrepo: $HOME/.small-dragonfly/dfdaemon/data/
```